app/uk/gov/hmrc/nisp/utils/Breadcrumb.scala

```diff
@@ -42,7 +42,8 @@ trait Breadcrumb {
       "voluntarycontribs" -> ((URLEncoder.encode(Messages("nisp.breadcrumb.nirecord.voluntaryContrib"), "UTF-8"), routes.NIRecordController.showVoluntaryContributions().url)),
       "gapsandhowtocheck" -> ((URLEncoder.encode(Messages("nisp.breadcrumb.nirecord.gapsandhowtocheck"), "UTF-8"), routes.NIRecordController.showGapsAndHowToCheckThem().url)),
       "exclusion" -> ((URLEncoder.encode(Messages("nisp.breadcrumb.excluded"), "UTF-8"), routes.ExclusionController.showSP().url)),
-      "exclusionni" -> ((URLEncoder.encode(Messages("nisp.breadcrumb.excluded"), "UTF-8"), routes.ExclusionController.showNI().url))
+      "exclusionni" -> ((URLEncoder.encode(Messages("nisp.breadcrumb.excluded"), "UTF-8"), routes.ExclusionController.showNI().url)),
+      "cope" -> ((URLEncoder.encode(Messages("nisp.breadcrumb.cope"), "UTF-8"), routes.AccountController.show().url))
```

Comment:
I know the customer will not see this URL but it should still have the cope one in to be correct.